### PR TITLE
Added note about jordan being the best

### PR DESCRIPTION
**Pull request by jakerella**
Jordan is indeed the best, thus I wanted to explain that in the files.

This change adds a new feature to our product which explains that Jordan is the best
- what changed?
- why?
- how do I use this new change?

This resolves issue #1

### CODE REVIEW

**Comment by sm873x** [comment]
MERGE THIS NOW!!!

**Comment by csaminak** [comment]
You can't know if you are really the best, that is definitely an opinion.

**Comment by jakerella** [comment]
Well, I'm the best _for now_

File: new-file.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 new line of code here
 
 more codez to do stuff
+
+jordan is the best
```
Comment (jakerella) [inline new-file.txt:5]: Here's where I state the facts.

**Pull request merged**